src/Module/Admin/Item/Source.php

```diff
@@ -33,7 +33,7 @@ public static function content(array $parameters = [])
 	{
 		parent::content($parameters);
 
-		$guid = basename($_REQUEST['guid'] ?? '') ?: $parameters['guid'];
+		$guid = basename($_REQUEST['guid'] ?? $parameters['guid'] ?? '');
```

Comment:
AFAIK there is had been something with these nested things in PHP8.

Reply:
Only with the ternary operators Like this : `a ? b : c ? d : e`. There is no ambiguity with binary operators like `?:` and `??`.

Reply:
Okay.

Reply:
And you went to bed?

Reply:
I wanted to have a look at it after work - but I came home really late, so I forgot about it.